### ISSUE TITLE
Unable to run lighthouse in Xvfb with tags/v2.0.0

### ISSUE THREAD

**Issue by cedricbellet** [GoogleChrome/lighthouse#2436]
I'm unable to run lighthouse in a docker container with chrome running in Xvfb with tag v2.0.0, whereas that works with tag v2.0.0-alpha.3. Has anything changed between the two tags that affects how lighthouse interacts with chrome?

With v2.0.0, ChromeLauncher waits for browser until it times out, seemingly failing to find the instance of google-chrome running in the Xvfb display.

```
$ node lighthouse-cli https://airhorner.com
>  ChromeLauncher Waiting for browser. +0ms
>   ChromeLauncher Waiting for browser... +2ms
>   ChromeLauncher Waiting for browser..... +513ms
>   ChromeLauncher Waiting for browser....... +504ms
>   ChromeLauncher Waiting for browser......... +503ms
>   ChromeLauncher Waiting for browser........... +502ms
>   ChromeLauncher Waiting for browser............. +503ms
>   ChromeLauncher Waiting for browser............... +502ms
>   ChromeLauncher Waiting for browser................. +503ms
>   ChromeLauncher Waiting for browser................... +502ms
>   ChromeLauncher Waiting for browser..................... +503ms
>   ChromeLauncher Waiting for browser....................... +501ms
> Unable to connect to Chrome
```

Chrome is up and running, having been launched with  `nohup google-chrome --no-first-run --disable-gpu --user-data-dir=$TMP_PROFILE_DIR --remote-debugging-port=9222 'about:blank' &`

**Comment by patrickhulce** [comment]
@cedricbellet there has been a breaking change were you will need to explicitly specify the port number as `--port 9222` if you want it to reuse an already open Chrome instance. It's probably trying to launch Chrome itself but is missing some necessary configuration for xvfb and fails. Does specifying the port fix your issue? Great data on xvfb issues though we should probably try to address it either way!

**Comment by ebidel** [comment]
I can confirm adding `--port=9222` fixes the issue with 2.0.

**Comment by ebidel** [comment]
@cedricbellet going to close this. Please reopen if setting the port doesn't work for you.
